onnxruntime/test/ir/flatbuffers_utils_test.fbs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// run flatc.exe from a debug build. Should be in <build output dir>/_deps/flatbuffers-build/Debug\flatc.exe
+// e.g. ..\..\..\build\Windows\Debug\_deps\flatbuffers-build\Debug\flatc.exe --cpp .\flatbuffers_utils_test.fbs
+// Diff the old and new generated header. Copy #include info from old as it has some customizations.
```

Comment:
ideally we can minimize the number of manual steps. there's already a [script](onnxruntime/core/flatbuffers/schema/compile_schema.py) for compiling the schemas. can that be used?